tools/generate-helpers/pg-table-bindings/store_test.go.tpl

```diff
@@ -73,6 +73,10 @@ func (s *{{$namePrefix}}StoreSuite) TestStore() {
 	s.False(exists)
 	s.Nil(found{{.TrimmedType|upperCamelCase}})
 
+    {{- /* Check for referenced schema */ -}}
```

Comment:
Why?

Reply:
We cannot unit test upserts for tables referencing other tables given our internal directory structure. For such cases, we first need to upsert the referenced object for respect referential constraint. Furthermore, our datastore structure is utterly variables and hence this cannot be autogenerated.

Reply:
Okay I see what you're saying, I ran into this on my PR as well.